examples/default/src/navigation/HomeStack.tsx

```diff
@@ -63,6 +65,11 @@ export const HomeStackNavigator: React.FC = () => {
         component={BasicComponentsScreen}
         options={{ title: 'Basic Components' }}
       />
+      <HomeStack.Screen
+        name="ComplexViews"
+        component={ComplexViewsScreen}
+        options={{ title: 'Basic Components' }}
```

Comment:
I think this screen title should be something like "Complex Views" instead of "Basic Components"

Reply:
Copy Paste 👎.